cgi-bin/Makefile

```diff
@@ -58,7 +58,7 @@ all:	$(TARGETS)
 # Make library targets...
 #
 
-libs:
+libs:	$(LIBTARGETS)
```

Comment:
Why? We no longer install libcupscgi as a shared library. The "libs" target is for installed libraries.